app/views/todos/index.html.erb

```diff
@@ -6,88 +6,9 @@
     <%= link_to "Edit Pivotal Token", edit_pivotal_path(current_user), class: "button" %>
   </div>
 
-  <% if [@my_pulls, @my_issues, @my_pivotal_stories].any?(&:present?) %>
-    <div>
-      <h3>Pull Requests (<%= @my_pulls.size %>): </h3>
-      <table class="table">
-        <thead>
-          <tr class="table__row table__row--head">
-            <th>Number</th>
-            <th>Title</th>
-            <th>State</th>
-            <th>Created</th>
-          </tr>
-        </thead>
-        <tbody>
-        <% @my_pulls.each do | pull | %>
-          <tr class="table__row">
-            <td><%= pull.pull_request_number %></td>
-            <td>
-              <%= link_to pull.repository_name, pull.repository_link, target: :_blank, class: "project_name"%><br/>
-              <%= link_to pull.title, pull.pull_request_link, target: :_blank %>
-            </td>
-            <td><%= pull.state %></td>
-            <td><%= created_at(pull) %></td>
-          </tr>
-        <% end %>
-        </tbody>
-      </table>
-    </div>
+  <%= render @my_pulls.present? ? "github_resource" : "empty_queue", resources: @my_pulls, queue_type: :pull_requests %>
```

Comment:
Just an idea here, instead of doing this conditional here, move it into the `_github_resource` partial.